src/sentry/shared_integrations/exceptions/__init__.py

```diff
@@ -171,8 +171,11 @@ class DuplicateDisplayNameError(IntegrationError):
 
 class IntegrationFormError(IntegrationError):
     def __init__(self, field_errors: Mapping[str, Any]) -> None:
-        super().__init__("Invalid integration action")
-        self.field_errors = field_errors
```

Comment:
Do we use this instance variable for anything?

Reply:
no

Reply:
it's not being helpful atm

Reply:
This is literally the only place it shows up in the code base.